arcgis-ios-sdk-samples/Content Display Logic/PortalItems.plist

```diff
@@ -130,6 +130,12 @@
 			<key>filename</key>
 			<string>SanFrancisco.tpkx</string>
 		</dict>
+		<dict>
+			<key>identifier</key>
+			<string>f5ff6f5556a945bca87ca513b8729a1e</string>
```

Comment:
The portal item: https://arcgis.com/home/item.html?id=f5ff6f5556a945bca87ca513b8729a1e